coalib/bears/requirements/NpmRequirement.py

```diff
@@ -24,3 +26,25 @@ def __init__(self, package, version=""):
         :param version: A version string. Leave empty to specify latest version.
         """
         PackageRequirement.__init__(self, 'npm', package, version)
+
+    def install_command(self):
+        """
+        Creates the installation command for the instance of the class.
+
+        >>> NpmRequirement('alex', '2').install_command()
+        'npm install alex@2'
+
+        :param return: A string with the installation command.
+        """
+        return "npm install {}@{}".format(self.package, self.version)
+
+    def is_installed(self):
+        """
+        Checks if the dependency is installed.
+
+        :param return: True if dependency is installed, false otherwise.
+        """
+        cmd = ['npm', 'show', self.package]
+        if platform.system() == 'Windows':
+            cmd = ['cmd', '/c'] + cmd
+        return not call_without_output(cmd)
```

Comment:
though it works, I've seen that shell expansion is still performed, which is something we want to avoid. After merge we need to file an issue :+1: (maybe we should use `npm.bat` instead (I hope it's a .bat file in npm like with perlcritic))